packages/common/driver-definitions/src/driverError.ts

```diff
@@ -145,6 +145,11 @@ export interface IDriverErrorBase {
 	 * See OnlineStatus enum in driver-utils package for expected values.
 	 */
 	online?: string;
+
+	/**
+	 * Whether service or socket was reachable and we got some error from them.
```

Comment:
"Whether service was reachable and we got some response from service."

I think it's more clearly describing it. If you want - we can clarify that response was with error code != 200